setup.py

```diff
@@ -103,11 +103,14 @@
 
 if dpcpp:
     DPCPP_CFLAGS = ['-D_DPCPP_']
-    DPCPP_LIBS = ['OpenCL', 'sycl', 'daal_sycl']
+    DPCPP_LIBS = ['OpenCL', 'sycl', 'onedal_sycl']
     if IS_LIN:
         DPCPP_LIBDIRS = [jp(dpcpp_root, 'linux', 'lib')]
     elif IS_WIN:
         DPCPP_LIBDIRS = [jp(dpcpp_root, 'windows', 'lib')]
+    if dal_build_version == (2021,7):
```

Comment:
maybe
dal_build_version  <= (2021,7)
?

Reply:
I tough about this - but not sure how  well  this work. 

in general all previous beta versions are obsolete and this code will be removed. together with previous versions

Reply:
so taking in mind time constraints - i would go with safe option )